CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] - ReleaseDate
 
+## [0.10.1] - 2023-12-11
+
```

Comment:
Doh, I forgot to fill this out, unfortunately the tag is already here, I'll create a separate follow on PR with the changelog info